lib/src/html_to_widgets.dart

```diff
@@ -14,6 +14,11 @@ import 'html_tags.dart';
 import 'pdfwidgets/bullet_list.dart';
 import 'pdfwidgets/number_list.dart';
 import 'pdfwidgets/quote_widget.dart';
+import 'dart:convert';
+
+RegExp base64RegExp = RegExp(
+  r'^(?:[A-Za-z0-9+/][A-Za-z0-9+/][A-Za-z0-9+/][A-Za-z0-9+/])*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=|[A-Za-z0-9+/]{4})$',
+);
 
```

Comment:
why need to use regex? detection of base64 is easy

Reply:
I used it because if HTML does not contain prefix data:image/